frontend/src/components/IngredientList.tsx

```diff
@@ -37,7 +37,9 @@ export default function IngredientList(props: IngredientListProps) {
                     }
                     </List>
                     <AddItem handleAddItem={addItem}/>
-                    <Button onClick={onTrigger}>Zutaten hinzufügen</Button>
```

Comment:
It is used to use titel "handle". I think, it is better to name the onlick function as handleOnlick.